lib/Service/OpenProjectAPIService.php

```diff
@@ -67,6 +67,8 @@ class OpenProjectAPIService {
 	public const AUTH_METHOD_OAUTH = 'oauth2';
 	public const AUTH_METHOD_OIDC = 'oidc';
 	public const MIN_SUPPORTED_USER_OIDC_APP_VERSION = '6.2.0';
```

Comment:
need to raise this one
```suggestion
	public const MIN_SUPPORTED_USER_OIDC_APP_VERSION = '7.0.0';
```